manifest.yaml

```diff
@@ -35,24 +23,12 @@ data:
   # Used to authenticate the culler to the hub. This string was generated with `openssl rand -hex 32`.
   auth.jhub-token.cull: 13fdff1305cd883e49223908186a63294922dadb59b5d1122473041f160c4b03
 ---
-apiVersion: v1
-kind: PersistentVolume
-metadata:
-  name: data8-hub-workdir-01
-spec:
-  capacity:
-    storage: 10Gi
-  accessModes:
-    - ReadWriteOnce
-  persistentVolumeReclaimPolicy: Retain
-  gcePersistentDisk:
-    pdName: data8-hub-workdir-01
-    fsType: ext4
----
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: data8-workdir
+  annotations:
+    volume.beta.kubernetes.io/storage-class: single-user-storage
```

Comment:
I see. Hmm... maybe I can call the storage class something more appropriate. Like gce-standard-storage, and use that for both the hub and single user. The storage class does not specify a size, so we can use the same storage class to provision pvc's that have different sizes. What do you think?